test/testdrive/introspection-sources.td

```diff
@@ -223,3 +223,18 @@ mv1_primary_idx mv1
     FROM mz_internal.mz_dataflow_operator_parents
     GROUP BY id, worker_id) counts
 1
+
+# Test that the roll-up of arrangement sizes is correct
+
+> CREATE TABLE t2 (a int)
+
+> INSERT INTO t SELECT * FROM generate_series(1, 100);
+
+> INSERT INTO t2 SELECT * FROM generate_series(1, 200);
+
+> CREATE VIEW vv2 AS SELECT sum(a) FROM t JOIN t2 USING (a);
+
+> CREATE DEFAULT INDEX ii2 ON vv2
+
+> SELECT records FROM mz_internal.mz_dataflow_arrangement_records WHERE name='ii2'
+303
```

Comment:
Do you know where the additional three is coming from? If it's due to timely internal, we should maybe make the test more robust by just checking if `records >= 300`.

Reply:
Two of them come from reduce rendering, the last is from the index itself.

Sure, I can change it to be >= 300.